SixTest/bin/SixTestWrapper.cpp

```diff
@@ -91,25 +115,25 @@ int main(int argc, char* argv[])
 	bool STFfail = false;
 	bool ExtraChecksfail = false;
 
-	if(atof(argv[2]) != 0)
+	if(atof(argv[4]) != 0)
 	{
 		fort10 = true;
 	}
 
-	if(atof(argv[3]) != 0)
+	if(atof(argv[5]) != 0)
 	{
 		fort90 = true;
 	}
 
-	if(atof(argv[4]) != 0)
+	if(atof(argv[6]) != 0)
 	{
 		STF = true;
 	}
 
-	if(atof(argv[5]) != 0)
+	if(atof(argv[7]) != 0)
 	{
 		CR = true;
-		KillTimes = ParseKillTimes(argv[6]);
+		KillTimes = ParseKillTimes(argv[8]);
 	}
 
```

Comment:
Extracting the executable names into a string here would be safer, in case the argv indices change.